docs/guides/securing-apps/mcp-authz-server.adoc

```diff
@@ -0,0 +1,113 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Integrating with MCP (Model Context Protocol)"
+priority=115
+summary="Using {project_name} as an authorization server in MCP">
+
+There are the following four MCP versions:
+
+* 2024-11-05 (initial version)
+* 2025-03-26
+* 2025-06-18
+* 2025-11-25 (latest version)
+
+The initial version is outdated, so this guide treats other three versions. The guide tells you the following:
+
+* Which MCP version {project_name} supports.
+* How to set up {project_name} as an authorization server in MCP.
+
```

Comment:
Maybe add a line that says this guide does not cover everything you need to know; and recommend reading the authorization section of the relevant MCP version as well?